include/openthread/tcp.h

```diff
@@ -215,6 +215,8 @@ struct otTcpEndpoint
     otTcpReceiveAvailable mReceiveAvailableCallback; ///< "Receive available" callback function
     otTcpDisconnected     mDisconnectedCallback;     ///< "Disconnected" callback function
 
+    uint32_t mTimers[4];
```

Comment:
remove hardcoding 4?

Reply:
```suggestion
    TimeMilli mTimers[4];
```

Reply:
I agree with @abtink. `TimeMilli` is nothing more than a wrapper around `uint32_t` with some methods added. So I just construct the `TimeMilli`s as I need them in the timer implementation.